contracts/HATVaults.sol

```diff
@@ -84,22 +80,16 @@ contract HATVaults is Claim, Deposit, Params, Pool, Swap, Getters, Withdraw {
     */
     function initialize(
         address _rewardToken,
-        uint256 _rewardPerBlock,
-        uint256 _startRewardingBlock,
-        uint256 _multiplierPeriod,
```

Comment:
shouldn't the reward controller be set on initialization?

Reply:
added this